docs/src/main/asciidoc/kubernetes-client.adoc

```diff
@@ -138,3 +138,85 @@ Note that to take advantage of these features, the `quarkus-test-kubernetes-clie
         <scope>test</scope>
     </dependency>
 ----
+
+== Note on implementing the Watcher interface
+
+Due to the restrictions posed by GraalVM, extra care needs to be taken when implementing a `io.fabric8.kubernetes.client.Watcher` if the application is to work in native mode.
```

Comment:
```suggestion
Due to the restrictions posed by GraalVM, extra care needs to be taken when implementing a `io.fabric8.kubernetes.client.Watcher` if the application is supposed to work in native mode.
```

Reply:
Or `intended`, dunno, looks like a verb is missing here :smile:

Reply:
Yup, that's a common trait when I write something - I skip the verb cause I'm thinking of the next phrase :P

Reply:
But actually this time the omission was on purpose. However since it threw you off as a reader, I will change it